grpc/build.gradle

```diff
@@ -18,7 +18,6 @@ dependencies {
     compile project(':conductor-common')
     compile project(':conductor-core')
 
-    protobuf 'io.chaossystems.grpc:grpc-healthcheck:1.0.+:protos'
```

Comment:
Why did we remove this?

Reply:
this protobuf was hosted at dl.bintray. dl.bintray no longer exists and it is not hosted anywhere else. Hence tried removing that from the project reference and still project build was successful. Hence, I believe, someone added this reference for trying out something and then forgot to remove its reference after the completion of the exercise.